release-notes-fetcher/config/release-notes-fetcher-camunda-8.5.yaml

```diff
@@ -1,6 +1,7 @@
 gitRef:
-  global: 8.5.6
-  operate: 8.5.5
-  identity: 8.5.4
-  tasklist: 8.5.4
-  zeebe: 8.5.6
+  global: 8.5.7
+  operate: 8.5.7
+  operate: 8.5.7
```

Comment:
❓ This looks suspiciously wrong.

Reply:
Thanks for pointing it out. Fixed via https://github.com/camunda/camunda-platform/pull/882/commits/e5345cd3a1393a9db7d49af690c83e9e87511ed2